packages/tocco-ui/src/StatedValue/StatedValue.js

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import ErrorList from '../FormField/ErrorList'
+import {
+  StyledStatedValueBox,
+  StyledStatedValueDescription,
+  StyledStatedValueError,
+  StyledStatedValueLabel,
+  StyledStatedValueWrapper
+} from './StyledStatedValue'
+import {stylingCondition} from '../utilStyles'
+
+const detectSignal = (dirty, hasError) => {
+  if (hasError) {
+    return stylingCondition.DANGER
+  } else if (dirty) {
+    return stylingCondition.WARNING
+  }
+}
+
+/**
+ * Wrap <EditableValue> and <FormattedVaule> in <StatedValue> to describe it
+ * and signal conditions.
+ */
+const StatedValue = props => {
```

Comment:
I dont know if i should like the name "StatedValue". Maybe we can discuss it. But isnt it just a new made "FormField"?

Reply:
FormFieldFloat